source/extensions/filters/http/cache/http_cache.h

```diff
@@ -170,6 +170,15 @@ using LookupResultPtr = std::unique_ptr<LookupResult>;
 // TODO(toddmgreer): Ensure that stability guarantees above are accurate.
 size_t stableHashKey(const Key& key);
 
+// The metadata associated with a cached response.
+// TODO(yosrym93): This could be changed to a proto if a need arises.
```

Comment:
for now, yes. However, it's worth noting that if a persistent cache is created using this code, and then you change it to a proto, you'd need to somehow invalidate the entire cache when referenced by the new code.

Maybe add that to the comment? Also how would we invalidate the cache? Would we put a version # into the key?

Reply:
I added this to the comment.

As for the how, IIUC, cache invalidation will need to be implemented at some point in the future anyway.

Reply:
For the scenario where we need to invalidate everything, we would alter the key generation--if there isn't anything more situation-specific, we might add a version #.